public/assets/styles/login.scss

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab');
+@import url('https://fonts.googleapis.com/css?family=Open Sans');
+
+$background-color: #f9f4f4;
+$facebook-color: #3b5998;
+$twitter-color: #00aced;
+$google-color: #f44d4d;
+$white: #ffffff;
+$site-color: #007fff;
+$line-color: #ced4da;
+
+%style-button {
+  border: 0;
+  font-size: 14px;
+  height: 50px;
+}
+
+%input-style {
+  border: 0;
+  border-bottom: 1px solid $line-color;
+  border-radius: 0;
+}
+
+.login-form{
+  background-color: $background-color;
+  border-bottom-left-radius: 10px;
+  border-top-left-radius: 10px;
+  padding: 80px 40px;
```

Comment:
in line with the teams conventions, we are to use em and not px for dimensions. Please resolve this.

Reply:
Done